### DISCUSSION TITLE
【2023.12.19】NEKOPARA  巧克力与香子兰全系列 (可能含有DLC)

### DISCUSSION THREAD

**Discussion by HUMHIHello** [pjy612/SteamManifestCache#93] [Share]
appid 与 keys :
Extra   899970
899971:a4d5d9b27ad9f7942a01fb69aa49a7864f74248ca2a03da836baf1a4824056eb

0 385800
385801:5dd3b7827073de8f0ae56600bfc3c5bd21048b189bb2b7187df8f9e43d0f3d2e

1  333600
333601:bdf5b06a88583b3fbbf04c0981445513402f6bf3829b481ae22c2fc94aaa2239
947650:b1041cb48540022c18ac837d59f947c12ae2e0ca81d764ecb2a6588a7d5b79e9

2   420110 
420111:9cf997ac955b6d9fe36be077adff4c110376a54dfdefc6977881ea3a920c1f15
947660:4569d5df4d3a3c91e6fa8a0aaa88868f7db832f060bdb8251cd89f277745cc40

3    602520
602521:26c6ec4aeeb2d7a5242153d94cc47f63b51cd10aee492803f8a8b32098098b05
947670:e6a0c5e1bbd515b8a3daf56b8157789b8489059a094a95a95893870180875e8f

4     1406990
1406991:3a788925e3104dac2a37bd06c1889d4de6ed05d62e672b9f9283f07fd45e3f4c"
[【2023.12.19】333600 NEKOPARA Vol.1 巧克力与香子兰.zip](https://github.com/pjy612/SteamManifestCache/files/13715024/2023.12.19.333600.NEKOPARA.Vol.1.zip)
[【2023.12.19】385800 NEKOPARA Vol.0 巧克力与香子兰.zip](https://github.com/pjy612/SteamManifestCache/files/13715025/2023.12.19.385800.NEKOPARA.Vol.0.zip)
[【2023.12.19】420110 NEKOPARA Vol.2 巧克力与香子兰.zip](https://github.com/pjy612/SteamManifestCache/files/13715026/2023.12.19.420110.NEKOPARA.Vol.2.zip)
[【2023.12.19】602520 NEKOPARA Vol.3 巧克力与香子兰.zip](https://github.com/pjy612/SteamManifestCache/files/13715027/2023.12.19.602520.NEKOPARA.Vol.3.zip)
[【2023.12.19】899970 NEKOPARA Extra 巧克力与香子兰.zip](https://github.com/pjy612/SteamManifestCache/files/13715028/2023.12.19.899970.NEKOPARA.Extra.zip)
[【2023.12.19】1406990 NEKOPARA Vol.4 巧克力与香子兰.zip](https://github.com/pjy612/SteamManifestCache/files/13715029/2023.12.19.1406990.NEKOPARA.Vol.4.zip)

刚刚解锁全系列 发现0 3 4 无法用工具解锁 看了一下是缺密匙了
这边顺便吧所有密匙扔出来了
我已是内测用户 仅为填充密匙库为目的

**Comment by Nobody2613** [comment]
Hero
